service-workers/service-worker/fetch-event-redirect.https.html

```diff
@@ -354,57 +318,6 @@
    'cors with credentials should succeed opaqueredirect interception ' +
    'and response should not be redirected');
 
-promise_test(function(t) {
-  return redirect_fetch_test(t, {
-    name: 'nonav-manual-nocors-redirects-to-sameorigin-creds',
-    redirect_dest: 'same-origin',
-    url_credentials: true,
-    expected_type: 'opaqueredirect',
-    expected_redirected: false,
-    request_init: {
-      redirect: 'manual',
-      mode: 'no-cors'
-    },
-    should_reject: false
-  });
-}, 'Non-navigation, manual redirect, no-cors mode Request redirected to ' +
-   'same-origin with credentials should succeed opaqueredirect interception ' +
-   'and response should not be redirected');
-
-promise_test(function(t) {
-  return redirect_fetch_test(t, {
-    name: 'nonav-manual-nocors-redirects-to-nocors-creds',
-    redirect_dest: 'no-cors',
-    url_credentials: true,
-    expected_type: 'opaqueredirect',
-    expected_redirected: false,
-    request_init: {
-      redirect: 'manual',
-      mode: 'no-cors'
-    },
-    should_reject: false
-  });
-}, 'Non-navigation, manual redirect, no-cors mode Request redirected to ' +
-   'no-cors with credentials should succeed interception ' +
-   'and response should not be redirected');
-
-promise_test(function(t) {
-  return redirect_fetch_test(t, {
-    name: 'nonav-manual-nocors-redirects-to-cors-creds',
-    redirect_dest: 'cors',
-    url_credentials: true,
-    expected_type: 'opaqueredirect',
-    expected_redirected: false,
-    request_init: {
-      redirect: 'manual',
-      mode: 'no-cors'
-    },
-    should_reject: false
-  });
-}, 'Non-navigation, manual redirect, no-cors mode Request redirected to ' +
-   'cors with credentials should succeed interception ' +
-   'and response should not be redirected');
-
```

Comment:
Why delete these test cases?  They are there to cover all possible inputs.  It seems we should just set the result to the correct expectation; which is to now reject.